### DISCUSSION TITLE
Run DAG at last day of month only with cron

### DISCUSSION THREAD

**Discussion by rsaleev** [apache/airflow#33996] [Q&A]
![изображение](https://github.com/apache/airflow/assets/21103312/b123c260-ae8b-474e-9602-60d305a3b37f)
I have created a schedule with cron , but DAG had not started, and it shows that next run is already scheduled on '2023-09-30 01:00:00'

Schedule: 0 1 L * * 

Next Run: 2023-08-31, 01:00:00

**Comment by potiuk** [comment]
Not sure what help you need? I think if you want to ask why it has not run before, hard to say because we do not see your dag.
